examples/agents/keyboard_agent.py

```diff
@@ -40,9 +42,9 @@ def rollout(env):
     human_wants_restart = False
     obser = env.reset()
     skip = 0
-    for t in xrange(ROLLOUT_TIME):
+    for t in range(ROLLOUT_TIME):
```

Comment:
This should not make any perf difference in Python 2, unless `ROLLOUT_TIME` is a few orders of magnitude larger than currently. (And in python 3 `range` is already a generator.)